lib/ansible/modules/service_facts.py

```diff
@@ -377,7 +377,7 @@ def get_info(self, name):
                 if variable == '' or '=' not in variable:
                     continue
                 else:
-                    k, v = variable.replace(undy, '', 1).split('=')
+                    k, v = variable.replace(undy, '', 1).split('=', 1)
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.